assets/js/components/Collection/Form.jsx

```diff
@@ -72,24 +79,20 @@ const CollectionForm = ({ collection }) => {
       <form onSubmit={handleSubmit(onSubmit)} data-testid="collection-form">
         <div className="columns">
           <div className="column is-two-thirds">
-            <div className="field">
-              <label htmlFor="collection-name" className="label">
-                Collection Name
-              </label>
-              <div className="control">
-                <input
-                  placeholder="Add collection Name"
-                  className={`input ${errors.collectioName ? "is-danger" : ""}`}
-                  type="text"
-                  ref={register({ required: true })}
-                  name="collectionName"
-                  id="collection-name"
-                  defaultValue={collection ? collection.name : ""}
-                  data-testid="input-collection-name"
-                />
-              </div>
-              <p className="help">Name of the Collection</p>
-            </div>
+            <UIFormFieldAndDisplay label="Collection Name">
+              <UIFormInput
+                placeholder="Add collection Name"
+                className={`input ${errors.collectionName ? "is-danger" : ""}`}
+                type="text"
```

Comment:
The component defaults to `type="text"`, so if it's a text input, just leave it off everywhere.

Reply:
cool!